docs/theming.md

```diff
@@ -0,0 +1,299 @@
+`headline6` | Headline variant 6
+`subtitle1` | Smaller than headline, reserved for medium-emphasis text that is shorter in length
+`subtitle2` | Subtitle variant 2
+`body1` | Used for long-form writing
+`body2` | Body variant 2
+`caption` | Used sparingly to annotate imagery
+`button` | A call to action used by different types of buttons
+`overline` | Used sparingly to introduce a headline
+
+For a given style, MWC supports the following custom properties (where `<STYLE>` is one of the style names, above):
+
+CSS custom property | Description
+--- | ---
+`--mdc-typography-font-family` | The base font-family
+`--mdc-typography-<STYLE>-font-family` | The font-family for `<STYLE>`. 
+`--mdc-typography-<STYLE>-font-size` | The font-size for `<STYLE>`. 
+`--mdc-typography-<STYLE>-line-height` | The line-height for `<STYLE>`.  
+`--mdc-typography-<STYLE>-font-weight` | The font-weight for `<STYLE>`. 
+`--mdc-typography-<STYLE>-letter-spacing` | The letter-spacing for `<STYLE>`. 
+`--mdc-typography-<STYLE>-text-decoration` | The text-decoration for `<STYLE>`. 
+`--mdc-typography-<STYLE>-text-transform` | The text-transform for `<STYLE>`. 
+
+For example, to set typeography properties for a button, you might use the following CSS:
+
+```css
+html {
+  --mdc-typography-button-font-size: 16px;
+  --mdc-typography-button-text-transform: none;
+}
+```
+
+
+**Example:** Overriding the global `font-family` property.
+
+This example shows loading a web font and setting it as the default font for MWC components.
+
+```html
+<link href="https://fonts.googleapis.com/css2?family=Arvo&display=swap" rel="stylesheet">
+
+<style>
+  html {
+    --mdc-typography-font-family: Arvo, serif;
+  }
+</style>
+```
+
+**Example:** Overriding the `font-family` property for `headline1` and `font-family` and `font-size` for `headline2`.
+
+
+```css
+html {
+  --mdc-typography-headline1-font-family: Arial, Helvetica, sans-serif;
+  --mdc-typography-headline2-font-family: Arial, Helvetica, sans-serif;
+  --mdc-typography-headline2-font-size: 3.25rem;
+}
+```
+
+## Shape
+
+MWC doesn't currently support shape customization.
+
+<!-- 
+Material components are all basically rectangular, but allow for some customization of the corner shape. 
+
+By default, components have a rounded corner. The default radius depends on the component's size 
+(small, medium, or large).
+
+MWC doesn't support cut corners.
+
+See also: [Material design shape](https://material.io/design/shape/applying-shape-to-ui.html#).
+-->
+
+## Iconography
+
+The [Material icons](https://material.io/resources/icons/) are a set of icons designed to meet the
+[Material icon design principles](https://material.io/design/iconography/system-icons.html#). 
+
+The MWC components use the Material icons by default, although you can choose to specify a different 
+icon font, or specify SVG icons inline for many components.
+
+To use the Material icons, you need to load the icon font:
+
+```html
+<link
```

Comment:
Any reason to have this split across multiline?

Reply:
Mmm no. It was copied from somewhere else and I retained the formatting. I can change that.